Makefile

```diff
@@ -79,6 +79,8 @@ openapi:
 # Build the docker image
 docker-build: all
 	hack/docker_build.sh
+docker-build-no-test: ks-apiserver controller-manager
+	hack/docker_build.sh
```

Comment:
What cases do we need to skip unit tests?

Reply:
It works well when just need to make a docker image on a personal computer quickly. Of course, it's not a good practice for using it.